src/containers/App/Content.tsx

```diff
@@ -189,9 +188,14 @@ function GetNodesList() {
     return null;
 }
 
-function GetCapabilities() {
-    capabilitiesApi.useGetClusterCapabilitiesQuery(undefined);
-    return null;
+function GetCapabilities({children}: {children: React.ReactNode}) {
```

Comment:
Now the whole interface is blocked by capabilities response

I think it would better to do it in background without impact on functionality that doesn't rely on capabilities (endpoint may timeout or throw an error)